ramp4-pcar4/storylines-editor
# Add modified property to dynamic panel schema, Fix SVG rendering

### Related Item(s)
#456 
#436

### Changes
- Add `modified` property to the dynamic panel schema
- Convert svgs to text rather than blob, so that they are correctly rendered

### Notes
- Regarding #436, I'll have to make a PR in the StoryRAMP repo to fix the issues with the preview

### Testing
Steps (#436):
1. Edit an existing product and change the logo to an svg (can use 
[git-bash](https://github.com/user-attachments/assets/770fe795-068c-4520-9c9c-55e5156ad728))
2. Load the product into `editor-main`
3. Click the `Edit Project Metadata` button, and observe that the svg logo is correctly displayed
4. Upload the svg to an image panel,  switch to a different slide and switch back to the image panel
5. Observe that the svg is correctly displayed

Steps (#456)
1. Make a new slide with a dynamic panel
2. Create an image panel and upload an image
3. Go to advanced editor and there should be no schema error
